handler/oauth2/strategy_jwt_test.go

```diff
@@ -55,7 +55,6 @@ var jwtValidCase = func(tokenType fosite.TokenType) *fosite.Request {
 			JWTClaims: &jwt.JWTClaims{
 				Issuer:    "fosite",
 				Subject:   "peter",
-				Audience:  []string{"group0"},
```

Comment:
Would this still work though? If so can we add a test to cover this case?

Reply:
No, it cannot work because we are filtering out `aud` from `GetExtraClaims`. Before the test was in fact buggy, I would claim, and didn't really check if audience is set in the generated (and tested) token. So setting this value does nothing.